src/log/log_slot.c

```diff
@@ -24,7 +24,7 @@ __log_slot_dump(WT_SESSION_IMPL *session)
     conn = S2C(session);
     log = conn->log;
     ret = __wt_verbose_dump_log(session);
-    WT_ASSERT(session, ret == 0);
+    WT_ASSERT_ALWAYS(session, ret == 0, "Failed to dump logging information");
```

Comment:
This is actually dumping information about the logging subsystem so I'd reword to: "Failed to dump log system information."